TeslaMateAgile/Services/BarryService.cs

```diff
@@ -23,7 +23,7 @@ public async Task<IEnumerable<Price>> GetPriceData(DateTimeOffset from, DateTime
     {
         var request = new BarryRequest
         {
-            Params = new string[] { _options.MPID, from.UtcDateTime.ToString("o"), to.UtcDateTime.ToString("o") }.ToList()
+            Params = new string[] { _options.MPID, from.UtcDateTime.ToString("s") + "Z", to.UtcDateTime.ToString("s") + "Z" }.ToList()
```

Comment:
What's the purpose of this change? It should already come out much like this with the "o" formatter

Reply:
They changed their API after they closed down. One could still ask for prices for older dates. Probably not the right spot for this as it has nothing to do with this PR.